usr.bin/gh-bc/Makefile

```diff
@@ -54,8 +54,8 @@ MAN_SRC_BC=	bc/A.1
 MAN_SRC_DC=	dc/A.1
 
 # prevent floating point incompatibilities caused by -flto on some architectures
-.if !${MACHINE_ARCH:Mmips*} && ${MACHINE_ARCH:Mpowerpc64*} && \
-    !${MACHINE_ARCH:Mriscv64*}
+.if !${MACHINE_ARCH:Maarch64*} && !${MACHINE_ARCH:Mmips*} && \
+    !${MACHINE_ARCH:Mpowerpc64*} && !${MACHINE_ARCH:Mriscv64*}
 CFLAGS+=	-flto
```

Comment:
At some point we should figure out (a) what breaks plain MIPS/RISC-V/PowerPC (b) what breaks CHERI...

Reply:
Upstream claims floating point is somehow involved for a).  I have hard time justifying effort on fixing LTO for bc(1) except for the fact that it may expose general issues preventing LTO on more useful applications.

Reply:
Oh indeed, I don't care about bc, just that if LTO doesn't work for bc then it's pretty useless.